### PR TITLE
Remove debug invocations from WebRender.

### PR DESCRIPTION
**Pull request by bholley**
This saves on codesize. See https://bugzilla.mozilla.org/show_bug.cgi?id=1430814

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/2350)

### CODE REVIEW

File: webrender/src/batch.rs [hunk 1]
```diff
@@ -1510,7 +1512,8 @@ impl ClipBatcher {
                                     ..instance
                                 });
                         } else {
-                            warn!("Warnings: skip a image mask. Key:{:?} Rect::{:?}.\n", mask.image, mask.rect);
+                            warn!("Warnings: skip a image mask");
```
[the remark this answers is not in the document]
Comment (bholley) [reply webrender/src/batch.rs]: By default, debug and trace are compiled out in release builds, while info, warn, and error are compiled in but disabled at runtime [1]. We could change this, but it's not obvious to me that we want to.

[1] https://searchfox.org/mozilla-central/rev/38bddf549db0f26d4d7575cf695958ff703a3ed7/toolkit/library/rust/shared/Cargo.toml#27

**Comment by Gankro** [comment]
Compiler isn't happy, build in debug mode to check your `debug!`s

Change otherwise seems reasonable, although a bit unfortunate.

**Comment by glennw** [comment]
This needs a rebase now, too.

**Comment by kvark** [comment]
Shouldn't we then remove `derive(Debug)` from those types, or do something like `cfg_attr(debug_assertions, derive(Debug))`?

**Comment by bholley** [comment]
The problem is that, in Rust, debug code still gets type-checked in release-mode builds. So any consumers of the Debug trait will break, even if they're theoretically debug-only.

**Pull request merged**